msu/hooks/states/world_state.nut

```diff
@@ -200,40 +200,93 @@
 	{
 		onBeforeSerialize(_out);
 		local meta = _out.getMetaData();
+		local modIDsString = ""
 		foreach (mod in ::MSU.System.Serialization.Mods)
 		{
 			meta.setString(mod.getID() + "Version", mod.getVersionString());
 			::MSU.Mod.Debug.printLog(format("MSU Serialization: Saving %s (%s), Version: %s", mod.getName(), mod.getID(), mod.getVersionString()));
 		}
+		foreach (mod in ::mods_getRegisteredMods()) modIDsString += mod.Name + ",";
```

Comment:
One-liner loop looks weird. Maybe I need to get used to it? Otherwise change into using curly braces.